src/app/feature/consultant/components/consultant-header-edit/consultant-header-edit.component.html

```diff
@@ -14,6 +14,23 @@ <h1 mat-dialog-title>Edit Personal Information</h1>
             <mat-error>Last Name is required</mat-error>
         </mat-form-field>
 
+        <mat-form-field>
+            <mat-label>Title</mat-label>
+            <input matInput formControlName="title" type="text">
+            <mat-error>Title is required</mat-error>
+        </mat-form-field>
+
+        <mat-form-field>
+            <mat-label>Practice</mat-label>
+            <mat-select formControlName = "practice" placeholder = "No Practice Selected">
+                <mat-option value = "Applications and Cloud Technology">Applications and Cloud Technology</mat-option>
```

Comment:
Do we want to use "and" or "&"? Right now the existing data has & so that is what the search uses. Ask Brianna because when I asked her about the search options she used "&"

Reply:
I think we've been using & for the most part so I guess for consistency sake & would be the way to go